lib/grammar2.pegjs

```diff
@@ -72,7 +71,13 @@ ctrl = ifctrl
      / forctrl
      / returnctrl
 
-ifctrl     = "if" _ c:innerstmt _ b:block { return p.parseIf(c, b) }
+ifctrl = "if" _ c:innerstmt __ b:block ei:(__ elseifcont)* {
+  ei = ei.map(function (pair) { return pair[1] })
```

Comment:
The `__` gobbles up whitespace, newlines, and any comments. So before that `map` happens `ei` will be an array similar to:

```js
[["\n# Comment\n", AST.If(...)], [" ", AST.If(...)]]
```

The `map` skips over the white-space globs and extracts just those `AST.If` nodes that correspond to each else-if condition.